electrical/PCB Assembly/README.md

```diff
@@ -0,0 +1,655 @@
+| Resistor 4.7K 1/4 Watt            | E7                 | R1                |
+| Resistor 10K 1/4 Watt             | E8                 | R4,6,8,10         |
+| Resistor 22K 1/4 Watt             | E9                 | R3,5,7,9          |
+| Resistor 10K 1/2 Watt             | E10                | R2                |
+| LM358 Op Amp                      | E25                | U1,2              |
+| DIP IC socket 8 Pos               | E33                | U1,2              |
+| Power Diode                       | E17                | D1                |
+| 10A Fuse                          | E16                | F1                |
+| USB A Connector                   | E34                | J12,14            |
+
+## Arduino Board References
+
+| **Component**                 | **Parts list Ref** | **Schematic Ref** |
+| ----------------------------- | ------------------ | ----------------- |
+| Term block 2P side entry      | E16                | J6                |
+| Connector Header pin 40P 40x1 | E15                | J2,3,4            |
+| Connector Header 16P 2x8      | E30                | J1                |
+| Connector Header 6P 6x1       | E14                | J5                |
+
+## Control Board Assembly
+
+### Motor & RoboClaw Connectors
+
+| **Item**              | **Ref** | **Qty** | **Image**                                                                       |
+| --------------------- | ------- | ------- | ------------------------------------------------------------------------------- |
+| OSR Control Board     | E1      | 1       | OSR Control Board<img src="/images/components/electronics/E1.png" height="100"> |
+| 6 Pos Side Term Block | E3      | 10      | <img src="/images/components/electronics/E3.png" height="100">                  |
+| 6 Pos Top Term Block  | E4      | 5       | <img src="/images/components/electronics/E4.png" height="100">                  |
+| 10 Pos Header Socket  | E5      | 5       | <img src="/images/components/electronics/E5.png" height="100">                  |
+| 5 Pos Header Socket   | E6      | 5       | <img src="/images/components/electronics/E6.png" height="100">                  |
+| Soldering Iron        | N/A     |         |                                                                                 |
+
+1. Begin by soldering the 6 Position Side entry terminal blocks **E3** into the
+   **top** side of the PCB, on the edge of the board as shown in the images
+   below. These terminal blocks will run motor power, encoder power, and encoder
+   signals between the motors/encoders and the RoboClaw motor controllers. The 6
+   terminal blocks will each be labeled with schematic reference designators
+   J17-J26 on the PCB. Be sure that the wire terminals face **OUTWARD** (away
+   from the center of the board) on all of these connectors.
+
+    ![Assembly Step 1](./../images/pcb_assembly/assembly/assembly_1.png)
```

Comment:
fix pics here and below (also make sure they conform to the new picture locations on master branch)